src/styles/popUpWindow.css

```diff
@@ -0,0 +1,165 @@
+.popup-window {
+  display: flex;
+  box-sizing: border-box;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  z-index: 9999;
+  pointer-events: all;
+  opacity: 0;
+  transform: rotate3d(1, 0.5, 0.5, 180deg) scale(0);
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.popup-window h3 {
+  font-size: 1.25rem;
+}
+
+.popup-window header {
+  flex-direction: column;
+  align-items: center;
+}
+
+.popup-window--visible {
+  opacity: 1;
+  transform: rotate3d(1, 0.5, 0.5, 0deg) scale(1);
+  transition: all ease-in 0.3s;
+}
+
+.popup-window--hidden {
+  opacity: 0;
+  transform: rotate3d(1, 0.5, 0.5, 180deg) scale(0);
+  transition: all ease-in 0.3s;
+}
+
+.popup-window__content {
+  position: relative;
+  width: 90%;
+  height: 90%;
+  background-color: var(--light-color);
+  border-radius: 0.5rem;
+  padding: 1rem;
+  box-shadow: 0 0.5rem 1rem rgba(0, 0, 0, 0.5);
+  pointer-events: all;
+  overflow-y: scroll;
+}
+
+.popup-window__close {
+  position: fixed;
+  top: 4%;
+  right: 6%;
+  font-size: 1.5rem;
+  color: white;
+  width: 33px;
+  border-radius: 20px;
+  cursor: pointer;
+  background-color: var(--primary-color);
+}
+
+.popup-window .comments-form {
+  gap: 0.5rem;
+  width: 100%;
+  max-width: 500px;
+}
```

Comment:
- [ ] Everything looks great. You did an amazing job in this project fulfilling almost all the project requirements impressively. Just for the sake of strictly complying with the design requirements, I request you to make a very slight change. The alignment of this component is not as in the original design. Just take this as a business requirement given be a client and make changes accordingly. :pray: 

![image](https://user-images.githubusercontent.com/64894927/142685833-2e8a16d4-60a2-414b-8e45-0310b48b47a1.png)

![image](https://user-images.githubusercontent.com/64894927/142686057-d38cec98-ce70-4e1e-9e3a-5ccb26414ff2.png)